vector/src/main/res/layout/fragment_create_room.xml

```diff
@@ -20,7 +20,7 @@
                 android:id="@+id/createRoomToolbar"
                 android:layout_width="match_parent"
                 android:layout_height="?actionBarSize"
-                app:title="@string/create_room_title" />
+                app:title="@string/direct_chats_header" />
```

Comment:
Are you sure about this change? It's a bit weird to have `direct_chats_header` for the create_room fragment

Reply:
This changement is made in Element: 
direct_chats_header = Conversations
create_room_title = New Room